packages/ckeditor5-build-trilium/src/config.ts

```
@@ -0,0 +1,152 @@
+import InternalLinkPlugin from './internallink';
+import MarkdownImportPlugin from './markdownimport';
+import CuttonotePlugin from './cuttonote';
+import IncludeNote from './includenote';
+import ReferenceLink from './referencelink';
+import indentBlockShortcutPlugin from './indent_block_shortcut';
+import removeFormatLinksPlugin from './remove_format_links';
+
+import {SpecialCharactersEmoji} from "./special_characters_emoji";
+
+// External plugins
+import Math from '@triliumnext/ckeditor5-math/src/math';
```

Biome (1.9.4)

[error] Do not shadow the global "Math" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)